website/.spelling

```diff
@@ -1099,7 +1099,7 @@ KinesisSupervisorIOConfig
 KinesisSupervisorTuningConfig
 Resharding
 resharding
-LZ4LZFuncompressedLZ4LZ4LZFuncompressednoneLZ4autolongsautolongslongstypeconcisetyperoaringcompressRunOnSerializationtruetypestreamendpointreplicastaskCounttaskCount
+LZ4LZFuncompressedLZ4LZ4LZFuncompressednoneLZ4autolongsautolongslongstypeconcisetyperoaringtypestreamendpointreplicastaskCounttaskCount
```

Comment:
hm, what is this i wonder? i know not new in this PR, but wondering did this file get messed up or something along the way? there are a few other lines like this in this section

Reply:
it has apparently always been like that https://github.com/apache/druid/pull/8548/files#diff-f0ad954629586689c64c31c7467c9ca21e5832971048aaa83f3b00532b7d05aaR643 :shrug: